### PR TITLE
handle folded block scalars in script blocks

### LINKED ISSUE #19: Handle folded block scalars in script blocks
Literal Style block scalars (`|`) are generally handled properly, but folded block scalars (`>`) confuse the syntax because each newline is treated as a newline in shell when it is actually an implicit line continuation.

This syntax is sometimes advantageous when using many arguments in a call and definitely preferrable from a syntax perspective over an implicit unqouted multi-line scalar or even a quoted multi-line scalar.

Syntax documentation: https://yaml.org/spec/1.2.2/#813-folded-style

Example (using GitLab syntax):

```yaml
release-finish:
  extends: release
  rules:
    - if: $CI_COMMIT_BRANCH =~ /^release\//
      when: manual
  script:
    - NEW_VERSION="${NEW_VERSION:-${CI_COMMIT_BRANCH#release/}}"
    - >-
      mvn
      --batch-mode
      $MAVEN_CLI_OPTS_DEFAULT
      $MAVEN_CLI_OPTS
      $MAVEN_GITFLOW_CLI_OPTS
      gitflow:release-finish
      -DpushRemote=true
      -DreleaseVersion="${NEW_VERSION}"
      -DskipTestProject=true
```

### PR DESCRIPTION
**Pull request by keith-hall**
fixes #19

### CODE REVIEW

File: AzureDevops.sublime-syntax [hunk 1]
```diff
@@ -93,6 +93,14 @@ contexts:
...
+    - meta_append: true
+    - match: ''
+      embed: scope:source.shell.bash.azure-devops
+      escape: (?!)
+
+  embedded-bash-folded:
     - meta_append: true
     - match: ''
       push: scope:source.shell.bash.azure-devops
+      with_prototype:
```
Comment (deathaxe) [inline AzureDevops.sublime-syntax]: This significantly grows compiled syntax's size.

Syntax cache grows from 0.8 to 6.1MB for the whole package.

Comment (deathaxe) [reply AzureDevops.sublime-syntax]: Not sure about the best way of a proposal, but I've created a PR to illustrate an alternative without such impact on syntax cache sizes.

Comment (keith-hall) [reply AzureDevops.sublime-syntax]: Thanks, I like the simplicity of the multiple extends :+1:

**Comment by deathaxe** [comment]
Are BitBucket pipelines relevant, too?

**Review by deathaxe — APPROVED** [review]

**Comment by keith-hall** [comment]
> Are BitBucket pipelines relevant, too?

I've never used BitBucket personally, but not as far as I can tell because scripts are always a single line sequence item as far as I have seen. If someone requests it in future we can add it then.

File: AzureDevops.sublime-syntax (hunk 1 above)
Comment (FichteFoll) [reply AzureDevops.sublime-syntax]: That's a rather elegant solution indeed.

**Pull request merged**